CHANGES.rst

```diff
@@ -1,6 +1,30 @@
 Changelog
 =========
 
+Version 1.1.0 [2024-08-14]
+--------------------------
+
+Features
+~~~~~~~~
+
+- Allowed ``GeoFeatureModelSerializer`` to support models without geometry field.
+
+Changes
+~~~~~~~
+
+**Dependencies:**
+
+- Bumped ``django-filters~=23.5``.
+- Added support for Django ``4.2``.
```

Comment:
Django 5 too :+1:

Reply:
I would say django 5.1 too?

Reply:
For sure, is it out yet?

Reply:
Yes, it was released only a few days ago
https://www.djangoproject.com/weblog/2024/aug/07/django-51-released/

Reply:
Indeed I remember having read that but I was not seeing it published in the list of releases on the website

Reply:
Thanks to @pandafy it's done, points remaining:

- https://github.com/openwisp/django-rest-framework-gis/pull/305
- update change log